README.md

```diff
@@ -224,7 +224,49 @@ For more information: https://pmd.github.io/pmd-5.4.1/customizing/rule-guideline
 
 - Flooded console output when running Checkstyle in debug mode in Maven  (- X ) - https://github.com/checkstyle/checkstyle/issues/3184;
 
+# ESH Guidelines Covered
+
+## A. Code Style
+1. The Java naming conventions should be used. - https://github.com/openhab/static-code-analysis/blob/master/src/main/resources/rulesets/checkstyle/rules.xml#L80
+2. Every Java file must have a license header. You can run mvn license:format on the root of the repo to automatically add missing headers. - https://github.com/openhab/static-code-analysis/blob/master/src/main/resources/rulesets/checkstyle/rules.xml#L34
+3. Every class, interface and enumeration should have JavaDoc describing its purpose and usage. - https://github.com/openhab/static-code-analysis/blob/master/src/main/resources/rulesets/checkstyle/rules.xml#L66
+4. Every class, interface and enumeration must have an @author tag in its JavaDoc for every author that wrote a substantial part of the file. - `Work in Progress`
+5. Every constant, field and method with default, protected or public visibility should have JavaDoc (optional, but encouraged for private visibility as well). - `Not covered yet`
+6. Code must be formatted using the provided code formatter and clean up settings. They are set up automatically by the official IDE setup. - `Not covered yet`
+7. Generics must be used where applicable. - `Not covered yet`
+8. Code should not show any warnings. Warnings that cannot be circumvented should be suppressed by using the @SuppressWarnings annotation. - `Not covered yet`
+9. For dependency injection, OSGi Declarative Services should be used. - `Work in Progress`
+10. Packages that contain classes that are not meant to be used by other bundles should have “internal” in their package name. - `Work in Progress`
+
+## B. OSGi Bundles
+1. Every bundle must contain a Maven pom.xml with a version and artifact name that is in sync with the manifest entry. The pom.xml must reference the correct parent pom (which is usually in the parent folder). - `Work in Progress`
+2. Every bundle must contain an about.html file, providing license information. - `Work in Progress`
+3. Every bundle must contain a build.properties file, which lists all resources that should end up in the binary under bin.includes. - `Work in Progress`
+4. The manifest must not contain any “Require-Bundle” entries. Instead, “Import-Package” must be used. - `Opened PR` (https://github.com/openhab/static-code-analysis/pull/19)
+5. The manifest must not export any internal package. - https://github.com/openhab/static-code-analysis/blob/master/src/main/resources/rulesets/checkstyle/rules.xml#L40
+6. The manifest must not have any version constraint on package imports, unless this is thoughtfully added. Note that Eclipse automatically adds these constraints based on the version in the target platform, which might be too high in many cases. - `Work in Progress`
```

Comment:
Well, we can add configuration properties for packages, that are usually imported with a version (like `org.junit`, `org.apache.commons`, `org.slf4j` and probably others) and these packages will not be considered a violation, even if they are imported with a version.

Also, if there is a bundle, that really needs to import a package with a version, it can be added to the suppressions.xml file (https://github.com/openhab/static-code-analysis/blob/master/src/main/resources/rulesets/checkstyle/suppressions.xml) and it will not be checked.